docker-entrypoint.sh

```diff
@@ -4,10 +4,7 @@
 # Copyright (C) 2025  Manuel G. Marciani
 # BSC-CNS - Earth Sciences
 
-dbus-daemon --config-file=/usr/share/dbus-1/session.conf --print-address --fork
-sleep 1
-/usr/bin/mysqld_safe &
-# TODO uncrappyfy this 
+/usr/bin/mariadbd-safe &
```

Comment:
There's a warning starting `mysqld_safe`, saying that's deprecated and to use `mariadb-safe` now. :+1: